### PR TITLE
Breadcrumbs: url of subfolder of hidden folder

### PR DESCRIPTION
**Pull request by ksuess**
folder hidden by IHideFromBreadcrumbs
See https://github.com/plone/Products.CMFPlone/issues/2935

### CODE REVIEW

**Comment by ksuess** [comment]
@jenkins-plone-org please run jobs

File: Products/CMFPlone/browser/navigation.py [hunk 1]
```diff
@@ -240,8 +240,8 @@ def breadcrumbs(self):
         if IHideFromBreadcrumbs.providedBy(context):
             return base
 
-        if base:
-            item_url = '%s/%s' % (base[-1]['absolute_url'], name)
+        # if base:
+        #     item_url = '%s/%s' % (base[-1]['absolute_url'], name)
```
Comment (jensens) [inline Products/CMFPlone/browser/navigation.py]: If code is no longer needed please just remove it. Git keeps it anyway, if someone is interested in it.

**Comment by ksuess** [comment]
@jenkins-plone-org please run jobs

**Comment by ksuess** [comment]
Thank you @jensens for reviewing. Would you please be so kind and review again, thank you.
A big thank you also to @petschki for your immediate response and hints!

**Comment by petschki** [comment]
could you please do exact the same for 5.1.x branch (new based on this branch) ? then it's fixed there too ... I'd suggest to add `[5.1]` in the PR title at the end ...

**Pull request merged**